client/src/use/validationHelpers.jest.spec.js

```diff
@@ -49,36 +49,111 @@ describe('test composables', () => {
         await result.$v.value.$touch();
         expect(result.hasErrorKey.value('one', 'required')).toBe(false);
     });
+});
 
-    test("applyExternalValidationErrors composition helper", async () => {
-        const { result } = mount(() => {
+describe("applyExternalValidationErrors composition helper", () => {
+    const factory = () => {
+        return mount(() => {
             const data = reactive({ one: '', two: '' });
             const eData = reactive({ one: [], two: [] });
 
-            const applyErrors = (error) => {
-                validationHelpers.applyExternalValidationErrors(data, eData, error);
+            const applyErrors = (error, strip = '') => {
+                return validationHelpers.applyExternalValidationErrors(data, eData, error, strip);
             }
             return { data, eData, applyErrors };
-        });
+        }).result;
+    };
+
+    test("single error strip prefix", async () => {
 
-        result.applyErrors({
+        const result = factory();
+        const returnValue = result.applyErrors({
             graphQLErrors: [
                 {
                     extensions: {
                         validation: {
-                            one: ['error1', 'error2']
+                            'user.one': ['error1', 'error2']
                         }
                     }
                 }
             ]
-        });
-
+        }, /^user\./);
+        expect(returnValue).toBe(true);
         expect(result.eData.one.length).toBe(2);
         expect(result.eData.two.length).toBe(0);
 
         result.data.one = 'one';
         await nextTick();
 
         expect(result.eData.one.length).toBe(0);
+
+    })
+    test("no validtion errors", () => {
+
+        const result = factory();
+        const returnValue = result.applyErrors({
+            graphQLErrors: [
+                {
+                    extentions: {
+                        authentication: 'Failed'
+                    }
+                }
+            ]
+        });
+
+        expect(returnValue).toBe(false);
+        expect(result.eData.one.length).toBe(0);
+        expect(result.eData.two.length).toBe(0);
+    })
+    test("multiple validation errors", async () => {
+        //Note: I don't think this occurs in the wild, but the structure suggetss that it may so... here we are.
```

Comment:
I don't find first person point of view in comments to be appropriate in open source. It's not understandable of who "I" refers to just by looking at the code. 
```suggestion
        // Note: This probably doesn't occur in the wild, but the structure suggests that it may, so... here we are.
```

Reply:
I feel like this is one of the useful things about blame, but I also agree with the suggestion.